### PR TITLE
split labels configurations add more prow commands

### PR DESCRIPTION
**Pull request by mmorel-35**
Thank you for contributing to Velero!

# Please add a summary of your change

* split `actions/labeler` and `jpmcb/prow-github-actions` configurations
* add more PROW commands

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [x] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [x] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

**Comment by kaovilai** [comment]
> Hi @blackpiglet,
> 
> It seems that `actions/labeler` and `jpmcb/prow-github-actions` configurations are incompatibles as kind is generating an error on `actions/label` execution.

Per their doc, [it never meant to be the same file](https://github.com/jpmcb/prow-github-actions#:~:text=Automatically%20label%20PRs,may%20look%20like%3A).

**Comment by blackpiglet** [comment]
I agree with @kaovilai.
I also made some changes in PR #7775.

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# This file is used by Auto Label PRs action.
```
Comment (blackpiglet) [inline .github/labels.yml:1]: Suggest to rename this file to `labeler.yaml`. Please also modify the action using this file.

https://github.com/actions/labeler?tab=readme-ov-file#usage

File: .github/workflows/prow-action.yml [hunk 2]
```diff
@@ -9,12 +9,21 @@ jobs:
   execute:
     runs-on: ubuntu-latest
     steps:
-      - uses: jpmcb/prow-github-actions@v1.1.3
+      - uses: jpmcb/prow-github-actions@v1
```
Comment (blackpiglet) [inline .github/workflows/prow-action.yml:12]: Sorry for the misleading, the action repo doesn't have a v1 tag.
We should still use v1.1.3

**Review by blackpiglet — APPROVED** [review]

File: .github/workflows/prow-action.yml [hunk 3]
```diff
@@ -11,10 +11,19 @@ jobs:
     steps:
       - uses: jpmcb/prow-github-actions@v1.1.3
         with:
-          # Only support /kind command for now.
           # TODO: before allowing the /lgtm command, see if we can block merging if changelog labels are missing.
-          prow-commands: "/area
-            /kind
+          prow-commands: |
+            /approve
```
Comment (kaovilai) [inline .github/workflows/prow-action.yml:16]: FYI approve would require an OWNERS file.
https://github.com/vmware-tanzu/velero/issues/7783

Comment (blackpiglet) [reply .github/workflows/prow-action.yml:16]: I suggest keeping the existing commands. I will create a new PR to introduce other PROW commands.
```
/area
/kind
/cc
/uncc
```

**Review by blackpiglet — APPROVED** [review]

**Review by kaovilai — APPROVED** [review]

**Review by qiuming-best — APPROVED** [review]

**Pull request merged**